runtime/__init__.py

```diff
@@ -110,17 +118,16 @@ def load_method(self, name: str) -> Method:
         Returns:
             The loaded method.
         """
-        return self._methods[name]
+        return self._methods.get(name, None)
 
 
 class OperatorRegistry:
     """The registry of operators that are available to the runtime.
-
-    Currently only supports printing out all registered operator names.
+    # TODO: Expose the kernel callables to Python.
     """
 
-    def __init__(self) -> None:
-        pass
+    def __init__(self, module: ModuleType) -> None:
+        self._legacy_module = module
```

Comment:
To be consistent with the rest of the file

```suggestion
    def __init__(self, legacy_module: ModuleType) -> None:
        self._legacy_module = legacy_module
```